embedding/cpp/elm_flutter_view.cc

```diff
@@ -49,9 +49,20 @@ bool ElmFlutterView::RunEngine() {
   evas_object_ =
       static_cast<Evas_Object *>(FlutterDesktopViewGetEvasObject(view_));
   if (!evas_object_) {
-    TizenLog::Error("Could not get an Evas object");
+    TizenLog::Error("Could not get an Evas object.");
     return false;
   }
+
+  evas_object_event_callback_add(
+      evas_object_, EVAS_CALLBACK_DEL,
+      [](void *data, Evas *evas, Evas_Object *object, void *event_info) {
+        auto engine = reinterpret_cast<FlutterDesktopEngineRef>(data);
+        if (engine) {
+          FlutterDesktopEngineShutdown(engine);
```

Comment:
Might be updated based on https://github.com/flutter-tizen/engine/pull/310.

Reply:
I changed it to add view_ destroy call to the destructor.